modules/nodes-cluster-enabling-features-console.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
```

Comment:
Should the full assembly path be here? `nodes/clusters/nodes-cluster-enabling-features.adoc`